### ISSUE TITLE
Might be reasonable to have a developer/debug/troubleshoot subcommand

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/stratis-cli#217]
Related to #138 but not the same.

These would consist of actions that don't necessarily interact w/ the daemon over the D-Bus but that might help a user get some information.

An example of an action that might fall into this category is checking whether stratisd would believe that a particular device has a stratisd signature. In order to implement this particular feature it would make sense to break some code out of stratisd into a separate library, or possibly expose it as a separate module.

Another possibility is to make this a separate application entirely, e.g., ```stratis-debug``` or something.

Such a choice leaves implementation choices pretty open. For example, it could be written in Rust, rather than Python, and thus make direct use of any sub-libraries that stratisd or devicemapper could make available.

**Comment by agrover** [comment]
Yeah, there seem to be a lot of little use cases that in aggregate may make this worth doing.
